<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayak GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCan_Log"/>
    <addaction name="actionCAN_Generator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCAN_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCan_Log">
   <property name="text">
    <string>Can Log</string>
   </property>
  </action>
  <action name="actionCAN_Settings">
   <property name="text">
    <string>CAN Settings</string>
   </property>
  </action>
  <action name="actionCAN_Generator">
   <property name="text">
    <string>CAN Generator</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
